packages/postcss-merge-longhand/src/__tests__/borders.js

```diff
@@ -431,9 +461,9 @@ test(
 );
 
 test(
-    'Should not convert currentColor (#559)',
+    'Should not convert currentcolor (#559)',
```

Comment:
Can we also add test with `currentColor` (some people use this writing style in code)?